website/static/playground.js

```diff
@@ -194,6 +196,9 @@ window.onload = function() {
 
 function setOptions(options) {
   OPTIONS.forEach(function(option) {
+    if (option === undefined) {
+      return;
+    }
```

Comment:
What's this check for? We're iterating over an array of known values, so there shouldn't be any undefined values here